schemas/catalog/20210202.json

```diff
@@ -0,0 +1,398 @@
+                                    "description": "Ethereum network that the artwork NFT exists on",
+                                    "minimum": 1,
+                                    "examples": [1, 4]
+                                },
+                                "contractAddress": {
+                                    "type": "string",
+                                    "description": "Address of the factory contract that was used to mint the visualizer NFT",
+                                    "examples": ["0xabEFBc9fD2F806065b4f3C237d4b59D9A97Bcac7", "0x7C2668BD0D3c050703CEcC956C11Bd520c26f7d4"]
+                                },
+                                "tokenId": {
+                                    "type": "integer",
+                                    "description": "Token ID of the artwork NFT",
+                                    "minimum": 0,
+                                    "examples": [69, 420]
+                                }
+                            },
+                            "additionalProperties": false
+                        }
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "signature": {
+            "type": ["object", "null"],
+            "description": "Information used to verify the authenticity of the record",
+            "examples": [
+                {
+                    "algorithm": "secp256k1",
+                    "encoding": "rlp",
+                    "publicKey": "0x0CcCcDAd491D8255d19475d4cC18c954AE185b0e",
+                    "signature": "0xd97ea5d9857f526537c3fed95eaedc22043fd530874beb41484902aa998cd36629b0fdd67fcceb6528f475aab018dc2994a9a5d06a2528aed0fdd0e68cfa4d781b"
+                }
+            ],
+            "required": [
+                "algorithm",
+                "encoding",
+                "publicKey",
+                "signature"
+            ],
+            "properties": {
+                "algorithm": {
+                    "type": "string",
+                    "description": "Algorithm used to sign the metadata body",
+                    "examples": ["secp256k1"]
+                },
+                "encoding": {
+                    "type": "string",
+                    "description": "Encoding used in the signature",
+                    "examples": ["rlp"]
+                },
+                "publicKey": {
+                    "type": "string",
+                    "description": "Public key used to sign the record metadata",
+                    "examples": ["0x0CcCcDAd491D8255d19475d4cC18c954AE185b0e"]
+                },
+                "signature": {
+                    "type": "string",
+                    "description": "The result of the public key signing the body of the record metadata. We sign the output of JSON.stringify(body), where body is the alphabatized JSON object of the records metadata.",
```

Comment:
`nit`: please add `minified` such that it reads `alphabetized and minified`.

Reply:
`nit`: the public key does not sign anything. Replace first part of description with `The result of the public key's corresponding private key signing the body of the record's metadata`

Reply:
`nit`: replace end with `where body is the alphabetized and minified JSON object retrieved from the body key in the record's metadata.`